components/api_request_helper/api_request_helper.h

```diff
@@ -17,12 +17,14 @@
 #include "base/time/time.h"
 #include "base/values.h"
 #include "net/traffic_annotation/network_traffic_annotation.h"
+#include "services/network/public/cpp/simple_url_loader.h"
```

Comment:
Can we move this `#includ` into .cc file? `SimpleURLLoader` is forward declared at the same time. If it's for using inner types like `OnResponseStartedCallback`, the forward declartion can be removed instead.

Reply:
The reason why I added that include was for inner types. I'll remove the forward decl and keep the include as per your suggestion.

Reply:
Done.